src/actions/commands/actions.ts

```diff
@@ -1647,6 +1647,13 @@ abstract class CommandFold extends BaseCommand {
   modes = [ModeName.Normal, ModeName.Visual, ModeName.VisualLine];
   commandName: string;
 
+  public doesActionApply(vimState: VimState, keysPressed: string[]): boolean {
+    // Don't run if there's an operator because the Sneak plugin uses <operator>z
+    return (
+      super.doesActionApply(vimState, keysPressed) && vimState.recordedState.operator === undefined
```

Comment:
Regardless of Sneak, I don't think the fold command works with an operator.

Reply:
If that's the case, then why did you need to add this check?

Reply:
Because currently the fold command runs even when there is an operator (but it shouldn't). So this is essentially fixing a bug, but one whose only effect is to conflict with Sneak.